docs/book/basic-usage.md

```diff
@@ -0,0 +1,43 @@
+# Basic Usage
+
+In the following example, a flash message is set in an edit action and the rendering is done after a redirect.
```

Comment:
Might be worth quoting "edit" in some way, as it's not normal text, but something that a user can do or carry out.

Reply:
You are right, 'in an "edit" action' or 'in a controller action' is much better.